### PR TITLE
TBI - Load sap icons in FLP sandbox

### PR DESCRIPTION
**Pull request by devinea**
Implements https://github.com/SAP/open-ux-tools/issues/1195

### CODE REVIEW

File: packages/ui5-application-writer/test/__snapshots__/options.test.ts.snap [hunk 1]
```diff
@@ -892,6 +892,24 @@ appDescription=Test App Description",
     };
 })(sap);
 
+function registerSAPFonts(){
+    var IconPool = sap.ui.require(\\"sap/ui/core/IconPool\\")
```
Comment (tobiasqueck) [inline packages/ui5-application-writer/test/__snapshots__/options.test.ts.snap:896]: Missing semicolon, or better, could we do it asynchronously? 
```
sap.ui.require(["sap/ui/core/IconPool"], function (IconPool) {  
   ...
});
```

**Review by tobiasqueck — CHANGES_REQUESTED** [review]

**Review by tobiasqueck — APPROVED** [review]
* changeset is correct
* code adheres to coding standards
* test snapshots correctly updated

I did not test it in an app because I didn't have a good example. Any recommendation what I could use for local testing?

**Comment by devinea** [comment]
@tobiasqueck Here is a sample app to test.
[sap_icons_poc1-2023-08-171500.zip](https://github.com/SAP/open-ux-tools/files/12370951/sap_icons_poc1-2023-08-171500.zip)

**Review by korotkovao — APPROVED** [review]

**Comment by devinea** [comment]
Thanks @tobiasqueck @korotkovao

**Pull request merged**